judging-algorithm/index.ts

```diff
@@ -0,0 +1,39 @@
+import { FinalOutputTables, HackerTeam, Judge, Room } from "./types";
+import { sortJudgesAndPeople } from "./formSchedule";
+import { parseHackerTeamCSV, parseJudgeCSV, parseRoomsCSV } from "./parser";
+import { convertHackersTablesToJson, convertJudgesTablesToJson } from "./formJsonOutput";
+
+// hardcode based on hackathon needs
+const allTimes: string[] = ['10:00', '10:15', '10:30', '10:45', '11:00', '11:15', '11:30', '11:45'];
```

Comment:
Just confirming, these are the final times PL decided on right

Reply:
140 hackers/3 -> 44 hackers if a team is 4 each, 50 / 4 -> 11 ish rounds per room at least
15 minute time slots do not work, just asked about it in PL